generate/src/contract/deployment.rs

```diff
@@ -50,8 +50,8 @@ fn expand_deployed(cx: &Context) -> TokenStream {
                 F: #ethcontract::web3::futures::Future<
                     Item = #ethcontract::json::Value,
                     Error = #ethcontract::web3::Error
-                > + Send + 'static,
```

Comment:
nit: `Future` does not need to be `Sync`.

Reply:
True. Meant to only change the Ts.